lmnet/lmnet/networks/classification/lmnet_v1.py

```diff
@@ -127,6 +127,7 @@ class LmnetV1Quantize(LmnetV1):
 
     def __init__(
             self,
+            quantize_first_convolution=True,
```

Comment:
This change not effect to classification model quantization or not. 

Please see this line.
https://github.com/blue-oil/blueoil/blob/master/lmnet/lmnet/networks/object_detection/lm_fyolo.py#L217

In future, I hope that to create new Mixin or base class which name is like `Quantizeable` which handle to quantization of all networks. but It takes some times to implement.

Our current stage, I think that step by step development is more productive, In this issue and PR, we only focus to pre process, ignore to handle to weight quantization. How do you think?

Reply:
As you said, create the `Quantizable` base class would make the maintenance and further implementation easier.

However, what do you mean by ignore weight quantization?
Do you mean we just change the preprocessor based on config file 
and apply weight quantization for all cases?

Reply:
As last conversation,
This change not effect to classification model quantization and it may takes several days to fix it,
In this issue and PR, don't need change classification model quantization.
To keep PR small changes makes productive, easy to test and review. 👍👍

Reply:
I already found the reason why the result is similar between two pre-processors.
It's because the value of `quantize_first_convolution`  in config file is `True/False` not `yes/no`.
So the if condition should be `if quantize_first_convolution` instead of `if quantize_first_convolution == "yes"`.
I already corrected the code and waiting for the training result.

Reply:
Is the `quantize_first_convolution` need ?